tools/accuracy_checker/configs/human-pose-estimation-0002.yml

```diff
@@ -1,27 +1 @@
-models:
-  - name: human-pose-estimation-0002
-
-    launchers:
-      - framework: dlsdk
-        allow_reshape_input: True
-        adapter:
-          type: human_pose_estimation_ae
-          heatmaps_out: heatmaps
-          nms_heatmaps_out: nms_heatmaps
-          embeddings_out: embeddings
-
-    datasets:
-      - name: ms_coco_val2017_keypoints
-
-        preprocessing:
-          - type: resize
-            size: 288
-            interpolation: LINEAR
-            aspect_ratio_scale: greater
-          - type: padding
-            stride: 32
-            pad_type: right_bottom
-
-        metrics:
-          - name: AP
-            type: coco_orig_keypoints_precision
+../../../models/intel/human-pose-estimation-0002/accuracy-check.yml
```

Comment:
here should be symlink on accuracy-check.yml, not text.
please create link or remove these files at all

Reply:
Thanks! I should have guessed. Windows is still playing tricks on me.